code/modules/events/ghost_role/pirates.dm

```diff
@@ -35,9 +36,10 @@
 	typepath = /datum/round_event/pirates/dutchman
 	weight = 0
 
-/datum/round_event_control/pirates/enclave
-	name = "Space Pirates - Imperial Enclave"
-	typepath = /datum/round_event/pirates/enclave
+/datum/round_event_control/pirates/nri
```

Comment:
do me a favor and modularize everything between these skyrat edit comments

Reply:
Spent a good while trying to and couldn't. This is horrible.